ios/browser/api/ledger/brave_ledger.mm

```diff
@@ -43,26 +43,41 @@
 #error "This file requires ARC support."
 #endif
 
-#define BLOG(verbose_level, format, ...)                  \
+namespace ledger {
+
+template <typename T>
+struct task_deleter {
```

Comment:
Is there a built-in Chromium thing that does this?

Reply:
Not as far as I'm aware of